readthedocs/config/config.py

```diff
@@ -839,11 +882,47 @@ def validate_submodules(self):
                 )
 
         with self.catch_validation_error('submodules.recursive'):
-            recursive = raw_submodules.get('recursive', False)
+            recursive = self.pop_config('submodules.recursive', False)
             submodules['recursive'] = validate_bool(recursive)
 
         return submodules
 
+    def validate_keys(self):
+        """
+        Checks that we don't have extra keys.
+
+        This should be called after all the validations are done.
+        """
+        self.pop_config('version', None)
+        wrong_key = '.'.join(self._get_extra_key(self.raw_config))
+        if wrong_key:
+            self.error(
+                wrong_key,
+                'Unsuported configuration: {}. Maybe a typo?'.format(wrong_key),
```

Comment:
Typo at `Unsuported`.

Also, perhaps `Unsupported configuration option: {}` for more specificity.

`Maybe a typo?` might not be clear for non-english speakers as well, so maybe something more formal here?

Reply:
That was the most formal sentence that I came up :disappointed:

Reply:
you honestly probably don't need the second portion of the error message. I think "unsupported configuration option: {x}" says enough

Reply:
Already changed in the last commit